public/styles/main_styles.css

```diff
@@ -100,4 +100,33 @@ footer {
 #signup-form p {
   margin-top: 30px;
   margin-left: 10px;
-}
\ No newline at end of file
+<<<<<<< HEAD
+}
+
+/*copied from signup **/
```

Comment:
instead of copying, you can even use the styling that's already written, just so we're not duplicating the same styling